src/index.html

```diff
@@ -10,8 +10,198 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+
+        <label>
+          Surname:
+          <input 
+            type="text" 
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label>
+          Name:
+          <input 
+            type="text" 
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <br>
+        <label>
+          How old are you?
+          <input 
+            type="number" 
+            name="age" 
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <br>
+        <label>
+          Full date of birth:
+          <input 
+            type="date" 
+            name="date"
+          >
+        </label>
+        <br>
+        <label>
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="acception"
+          >
+        </label>
+      </fieldset>
+    </form>
+
+    <form>
+      <fieldset>
+        <legend>
+          Registration
+        </legend>
+
+        <label>
+          E-mail:
+          <input 
+            type="email" 
+            name="email"
+            placeholder="email@example.com"
+            multiple
+            autocomplete="off"
+            required
+          >
+        </label>
+        <br>
+        <label>
+          Password:
+          <input 
+            type="password" 
+            name="password"
+            minlength="4"
+            maxlength="12"
+            autocomplete="off"
+            required
+          >
+        </label>
+      </fieldset>
+    </form>
+
+    <form>
+      <fieldset>
+        <legend>
+          An interersting fact about you?
+        </legend>
+        <label>
+          Do you love cats?
+          <input
+            type="radio"
+            id="catsChoiseYes"
+            name="love"
+            value="yes"
+          >
+          <label 
+            for="catsChoiseYes"
```

Comment:
the same
Fix in all cases
![image](https://user-images.githubusercontent.com/91411692/230862167-6e59a390-05ea-4d0c-8942-6368db8c5d78.png)